pt-BR/mail/chrome/messenger-newsblog/feed-subscriptions.dtd

```diff
@@ -1,5 +1,5 @@
 <!-- Subscription Dialog -->
-<!ENTITY newsBlogSubscriptions.label "RSS Subscriptions">
+<!ENTITY newsBlogSubscriptions.label "Assinaturas RSS">
```

Comment:
I'm going to change to Inscrições RSS to maintain consistency with what is currently on the current version, even though in PTBR, the usual way to say it is Assinaturas RSS:

https://www.microsoft.com/Language/en-US/Search.aspx?sString=rss-subscriptions&langID=pt-br